### PR TITLE
Apply MVP Structure

### PR DESCRIPTION
**Pull request by DungTheRipper**
## Related Tickets
- [#Task 78170: Apply MVP Structure](https://edu-redmine.sun-asterisk.vn/issues/xxxx)
## WHAT
- Apply MVP structure into the project
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/WtYPAy |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

**Review by Hoaivuxuan — APPROVED** [review]

**Review by ireone01 — APPROVED** [review]

File: app/src/main/res/layout/fragment_detail.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    android:layout_height="match_parent"
+    tools:context=".screen.detail.DetailFragment">
+
+    <!-- TODO: Update blank fragment layout -->
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="@string/hello_blank_fragment" />
+
+</FrameLayout>
```
Comment (ghost) [inline app/src/main/res/layout/fragment_detail.xml]: End of content must be a empty line

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    android:layout_height="match_parent"
+    tools:context=".screen.home.HomeFragment">
+
+    <!-- TODO: Update blank fragment layout -->
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="@string/hello_blank_fragment" />
+
+</FrameLayout>
```
Comment (ghost) [inline app/src/main/res/layout/fragment_home.xml]: empty line
```suggestion
</FrameLayout>

```

File: app/src/main/res/layout/fragment_search.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    android:layout_height="match_parent"
+    tools:context=".screen.search.SearchFragment">
+
+    <!-- TODO: Update blank fragment layout -->
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="@string/hello_blank_fragment" />
+
+</FrameLayout>
```
Comment (ghost) [inline app/src/main/res/layout/fragment_search.xml]:
```suggestion
</FrameLayout>

```

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">Android-template</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
 </resources>
```
Comment (ghost) [inline app/src/main/res/values/strings.xml]:
```suggestion
</resources>

```

**Review by ghost — CHANGES_REQUESTED** [review]
Check my cmts

**Review by ghost — APPROVED** [review]
lgtm

**Pull request merged**